python-sdk/nuscenes/eval/detection/loaders.py

```diff
@@ -56,11 +56,15 @@ def load_gt(nusc, eval_split, cfg) -> EvalBoxes:
             if detection_name is None:
                 continue
 
-            # Get attribute_labels.
-            attribute_labels = np.zeros((len(cfg.attributes),), dtype=bool)
-            for i, attribute in enumerate(cfg.attributes):
-                if attribute_map[attribute] in sample_annotation['attribute_tokens']:
-                    attribute_labels[i] = True
+            # Get attribute_name.
+            attr_tokens = sample_annotation['attribute_tokens']
+            attr_count = len(attr_tokens)
+            if attr_count == 0:
+                attribute_name = ''
+            elif attr_count == 1:
+                attribute_name = attribute_map[attr_tokens[0]]
+            else:
+                raise Exception('Error: GT annotations must not have more than one attribute!')
```

Comment:
Is this exception already covered in the verify script?

Reply:
Yes, I added it now.